packages/amplify-cli/src/input-manager.ts

```diff
@@ -197,5 +199,10 @@ export function verifyInput(pluginPlatform: PluginPlatform, input: Input): Input
 function aliasArgs(argv: string[]) {
   if (argv.length >= 4 && argv[2] === 'override' && argv[3] === 'project') {
     argv[3] = 'root';
+
+    // Also update gitignore to latest list - mainly to exclude amplify/backend/awscloudformation dir from .gitingore for older projects
+    const { projectPath } = stateManager.getLocalEnvInfo();
+    const gitIgnoreFilePath = pathManager.getGitIgnoreFilePath(projectPath);
```

Comment:
will this include build files also inside awscloudformation?

Reply:
Nope. build/ folders are excluded. Checked on it.